src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,83 @@
+package core.basesyntax.service;
+
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.exceptions.RegistrationException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class RegistrationServiceImplTest {
+    private static RegistrationService registrationService;
+    private static StorageDaoImpl storageDao;
+    @BeforeAll
+    static void beforeAll() {
+        registrationService = new RegistrationServiceImpl();
+        storageDao = new StorageDaoImpl();
```

Comment:
You are using the `@BeforeAll` annotation here, which means the `beforeAll` method will be executed only once before all test cases. This might lead to problems if your tests are not independent, as the state of `registrationService` and `storageDao` will not be reset before each test. Consider using `@BeforeEach` to ensure that a fresh instance of `registrationService` and `storageDao` is used for each test.